coalib/results/Result.py

```diff
@@ -108,15 +120,19 @@ def from_values(cls,
                     debug_msg='',
                     diffs: (dict, None)=None,
                     confidence: int=100,
-                    aspect: (aspectbase, None)=None):
+                    aspect: (aspectbase, None)=None,
+                    message_arguments: (dict, list)={},
+                    ):
```

Comment:
yeah. I didn't want to change the argument message to message_base because I feared it would break some bear.